examples/fastify/index.js

```diff
@@ -0,0 +1,39 @@
+const fastify = require('fastify')({logger: false})
+const Honeybadger = require("../../dist/server/honeybadger.js")
+Honeybadger.configure({});
+
+fastify.addHook('preHandler', Honeybadger.requestHandler);
+
+fastify.setErrorHandler((err, req, reply) => Honeybadger.withRequest(req, () => {
+    Honeybadger.notify(err)
+    reply.send("NOT OK")
+}));
+
+let reqId = 0;
+fastify.get('/test-context', function (request, reply) {
```

Comment:
Since this place is for _examples_, I am thinking that the code should be focused on setting up the framework with Honeybadger. This example is the setup + a test for an internal implementation detail (isolated context). I think we could keep it simpler and have 2 routes:
```
fastify.get('/unhandled', function (request, reply) {
  throw new Error('Unhandled error. Should be reported on Honeybadger dashboard');
})

fastify.get('/report', function (request, reply) {
  Honeybadger.notify('Hello World!')
  return 'Message should have been reported to Honeybadger! Please check your Honeybadger dashboard.'
  throw new Error('Unhandled error. Should be reported on Honeybadger dashboard');
})
```

What do you think?

Reply:
Done @subzero10